velox/exec/RowContainer.cpp

```diff
@@ -560,7 +574,8 @@ void RowContainer::extractSerializedRows(
     }
   }
 
-  size_t totalBytes = nullBytes * rows.size() + fixedWidthRowSize * rows.size();
+  size_t totalBytes =
```

Comment:
const totalBytes?

Reply:
totalBytes is updated to include the size of variable length fields a few lines below this (I didn't have to update that logic as part of this change so it doesn't appear in the diff context).